bindata/network-diagnostics/001-rbac.yaml

```diff
@@ -0,0 +1,77 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: network-diagnostics
+  namespace: openshift-network-diagnostics
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: network-diagnostics
+rules:
+- apiGroups: [""]
+  resources:
+  - endpoints
+  - namespaces
+  - pods
+  - services
+  - nodes
+  - secrets
```

Comment:
should it really be able to access all secrets across all namespaces? I'd rather not if possible...

Reply:
I will get back to you on this, but I had to add that as I was getting issues from library-go connectivity check controller.
Will remove, run it and put a gist the thing I got, can't remember off the top of my head.

Reply:
Still need to remove this.

Reply:
So you suggest having a Role with just secrets, so it's bound to openshift-network-diagnostics namespace and then bind it to the network-diagnostics ServiceAccount?

Reply:
Yes, a Role rather than a ClusterRole is the right thing to do here.

We **definitely** don't want to have full access to all secrets across all namespaces.

Reply:
++

Reply:
done